### PR TITLE
Added a root to routes and set up a default just in case

### PR DESCRIPTION
**Pull request by Radatat**

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -1,9 +1,10 @@
 Rails.application.routes.draw do
-  # The priority is based upon order of creation: first created -> highest priority.
-  # See how all your routes lay out with "rake routes".
 
-  # You can have the root of your site routed with "root"
-  # root 'welcome#index'
+  root 'games#index'
+
+  # If no routs are found, use this as the default
+  match ':controller(/:action(/:id))', :via => :get
```
Comment (YusufSoomro) [inline config/routes.rb]: Did you add this in?

Comment (Radatat) [inline config/routes.rb]: Yasss

Comment (YusufSoomro) [inline config/routes.rb]: What does this do?

Comment (Radatat) [inline config/routes.rb]: If the user goes to a route but doesn't supply the full path, this line intuitively goes through and finds a match that fits the Controller#Action pattern

Comment (YusufSoomro) [inline config/routes.rb]: Ok, you're cool to merge then.

**Pull request merged**